### ISSUE TITLE
Create a binary release

### ISSUE THREAD

**Issue by sobolevn** [wemake-services/dotenv-linter#160]
Some users say, that we need a binary release, so it would be possible to just use `wget` or `curl` to install this project.

So, let's do it.

**Comment by sobolevn** [comment]
Tools:
- http://www.pyinstaller.org/
- https://nuitka.net/

**Comment by sobolevn** [comment]
https://tryexceptpass.org/article/distributing-python-applications/

**Comment by Khabib73** [comment]
It's time to learn Rust and rewrite the project from Python :)

**Comment by sobolevn** [comment]
nah, never mind, there's https://github.com/dotenv-linter/dotenv-linter that does it already :)
